poetry/packages/package.py

```diff
@@ -162,6 +162,13 @@ def _get_author(self):  # type: () -> dict
 
         m = AUTHOR_REGEX.match(self._authors[0])
 
+        if m is None:
+            logger.warning(
+                "Invalid author string. Must be in the format: "
+                "John Smith <john@example.com>"
+            )
+            return {"name": None, "email": None}
+
```

Comment:
Hello @yamagen0915,

thanks a lot for your contribution!

I think instead of just returning None values, we should raise a ValueError with your descriptive message. I'm afraid many users will not read the console output and will wonder why their author isn't in the metadata. So we should follow the principal of "fail fast".

fin swimmer

Reply:
@finswimmer 
Thanks for the review!
I have fixed. 6a0ac4a